### PR TITLE
create main activity layout and home fragment layout

### PR DESCRIPTION
**Pull request by nguyenvanducc**
https://edu-redmine.framgia.vn/issues/1542?issue_count=38&issue_position=6&next_issue_id=1543&prev_issue_id=1541

### CODE REVIEW

File: app/src/main/java/com/framgia/nguyenvanducc/soundcloud33/screen/main/MainActivity.java [hunk 1]
```diff
@@ -1,15 +1,40 @@
...
+import android.view.MenuItem;
+import android.widget.TextView;
 
 import com.framgia.nguyenvanducc.soundcloud33.R;
 import com.framgia.nguyenvanducc.soundcloud33.screen.BaseActivity;
 
-public class MainActivity extends BaseActivity {
+public class MainActivity extends BaseActivity implements
+        BottomNavigationView.OnNavigationItemSelectedListener, MainContract.View {
+    private TextView mToolbarTitle;
```
Comment (tuannguyen38) [inline app/src/main/java/com/framgia/nguyenvanducc/soundcloud33/screen/main/MainActivity.java]: Convention 3.1

File: app/src/main/res/values/colors.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="colorPrimary">#3F51B5</color>
-    <color name="colorPrimaryDark">#303F9F</color>
-    <color name="colorAccent">#FF4081</color>
+    <color name="colorPrimary">#9c27b0</color>
+    <color name="colorPrimaryDark">#6a0080</color>
+    <color name="colorAccent">#f06292</color>
```
Comment (tuannguyen38) [inline app/src/main/res/values/colors.xml]: 3 4 5 convention 3.2

**Comment by tuannguyen38** [comment]
ready

**Comment by nguyenvanducc** [comment]
ready

**Pull request merged**